### PR TITLE
[issue 55] Poor UX when trying to create an issue if the repository does not permit it #55 

### LINKED ISSUE #55: Poor UX when trying to create an issue if the repository does not permit it
If a repo in bitbucket has issue creation disabled, the plugin UX does not handle this gracefully.
We show an error message in the modal that should instead show a non-technical message to explain issue creation is not allowed keeping only the text from the message.
image

Steps:
- Login to your Bitbucket org such as https://bitbucket.org/DHaussermann
- Setup and configure the plugin oAuth using the instructions in the read-me
- Connect a test user
- Select any repo in the organization and disable issue creation
- Make or locate a post
- Select the ... menu and _Create Bitbucket Issue_
- Select the repository from the dropdown where issue creation has been disabled 
- Click Submit

**Observed**: `{"id":"","message":"failed to create issue: Sorry, either you don't have access to the repo DHaussermann/helloworld with the user dkh-sysadmin or it is no longer available","status_code":404}`

### PR DESCRIPTION
**Pull request by sibasankarnayak**
Improved UX by handling error properly

current UX

image

ticker here
Fixes #55

### CODE REVIEW

File: webapp/src/utils/user_utils.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export function getErrorMessage(str) {
+    try {
+        const parsed = JSON.parse(str);
+        return parsed.message;
+    } catch (e) {
+        return str;
+    }
+}
```
Comment (hanzei) [inline webapp/src/utils/user_utils.js]: Nit: Missing line break

File: webapp/src/utils/user_utils.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export function getErrorMessage(str) {
```
Comment (hanzei) [inline webapp/src/utils/user_utils.js]: Why is the file called `user_utils.js`?

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
@DHaussermann Can we test this PR as part of release testing?

**Review by DHaussermann — APPROVED** [review]
This task can be tested post merge as part of release testing.

LGTM!

**Pull request merged**